developersociety/commonslibrary
# Do no include tags in search

# Sources
https://trello.com/c/4z05fssx/19-be-tag-remapping-script

# Description
Do no include tags titles as searchable

# Setup instructions
`fab get_backup`

# How to test
1. Go and login and go to http://127.0.0.1:8000/ type `fundraising` and it should return 39 resources click to only include the `Fundraising` tag and the results should be as 29
